drift/lib/src/runtime/manager/composable.dart

```
@@ -42,6 +42,11 @@ class JoinBuilder {
     return currentColumn.hashCode ^ referencedColumn.hashCode;
   }
 
+  @override
```

I've missed this in the previous PR. But while we're at it, can we change `hashCode` to use `Object.hash` instead of xors? I think `==` should also not be covariant and instead do a `other is JoinBuilder` check - otherwise it throws instead of returning false which is pretty weird.